docs/core/extensions/globalization-icu.md

```diff
@@ -253,7 +253,25 @@ For framework-dependent apps (not self-contained) where ICU is consumed from a l
 
 This must be done for all the ICU binaries for the supported runtimes. Also, the `NuGetPackageId` metadata in the `RuntimeTargetsCopyLocalItems` item group needs to match a NuGet package that the project actually references.
 
-### macOS behavior
+## Loading Specific ICU Version on Linux
+
+By default, when using ICU on Linux, .NET attempts to load the latest installed version of ICU from the system. However, you can specify a specific version of ICU to load by setting the `DOTNET_ICU_VERSION_OVERRIDE` environment variable. Here's how it works:
```

Comment:
The `build` setting is going to map to some very old ICU version and it is going change from major version to major version.

When would a .NET user want to use the `build` setting? Should it even be documented?

Reply:
We may want to delete it from the sources if we are not able to tell what it is for.

Reply:
I was documenting the current behavior but realized I'm unsure why this value was introduced in the first place, as it doesn't seem generally beneficial. I'll remove it from the documentation to prevent users from relying on it.

Reply:
> We may want to delete it from the sources if we are not able to tell what it is for.

I think this is done to support the scenario mentioned in https://github.com/dotnet/runtime/pull/79251#issuecomment-1338384561.  

@wscho77 do you still depend on setting `DOTNET_ICU_VERSION_OVERRIDE` to `build` value? if so, could you please elaborate more on your scenario and how this helps? and what would be the problem if not using it.

Reply:
@tarekgh We are still using that setting. In the case of CE devices, many I/O operations  occur during booting time, causing I/O related delays. This causes a lot of delays in app running at this time.
In the case of ICU loading, dlopen() is called multiple times to find the latest version, which causes app execution to slow down. This setting allows us to reduce delays caused by unnecessary I/O operation.

Reply:
But you can specify the version number and not use `Build` as a value for the environment variable. right?